static/css/style.css

```diff
@@ -1,8 +1,22 @@
 :root {
     --bg-color: #FAFAFA;
+    --border-color: #ddd;
     --text-color: #282828;
     --primary-color: #2980b9;
     --primary-text-color: #FFF;
+    --secondary-text-color: #505050;
+    --button-hover-color: #f0f0f0
+}
+@media screen and (prefers-color-scheme: dark) {
```

Comment:
I really like the addition of a darkmode, but my eyes are hurt by the inverted black/white contrast the same way as if I used a light theme. I prefer colors like #e3e0d7 for text and #242424 for background. I'll open a follow-up issue.